build-pipeline/vg-pipeline-module-pr.yml

```diff
@@ -97,7 +98,7 @@ stages:
         - task: replacetokens@3
           displayName: "Token Replace .env.docker"
           inputs:
-            targetFiles: './vanguard-api/.env.docker'
+            targetFiles: './vanguard-api/.env.docker => .env'
```

Comment:
this is important because in the earlier code, it changed the `.env.docker` file which is tracked in git. and when `git status` has any change, the vg-api doesn't pull pre-existing images from DH. rather re-builds all services.
And the time difference in these two scenarios is <1min vs 4+mins. so we shouldn't make any un-necessary change on vg-api repo in pipeline as that could save us build-time

cc: @rajatsharma111